libs/sdk-model/src/dashboard/filterContext.ts

```diff
@@ -47,6 +47,18 @@ export interface IDashboardAttributeFilterParent {
  */
 export type DashboardAttributeFilterSelectionMode = "single" | "multi";
 
+/**
+ * Item that is used to validate availability of attribute element.
+ *
+ * The item is either metric or is converted to metric (in case of attribute, label, or fact) by backend
+ * and cross-tabulation report is executed to determine if attribute element should be available (visible).
+ *
+ * @alpha
+ */
+export interface IElementValidationItem {
+    identifier: ObjRef;
```

Comment:
would not be ref, itemRef or something similar better? "identifier" implies limited usage to idRef, which is probably true, but should not leak to the model level

Reply:
ok, thx for explanation. I had no idea about this automagic